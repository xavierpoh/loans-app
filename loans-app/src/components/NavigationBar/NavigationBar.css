.nav-container {
  background-color: #fff;
  border-bottom: 1px solid #EBEBEB;
}

.navbar-brand img {
  height: 40px;
  width: auto;
  bottom: 6px;
  position: absolute;
}