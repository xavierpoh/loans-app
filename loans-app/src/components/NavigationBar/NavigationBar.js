import React from 'react';
import { Navbar } from 'react-bootstrap';
import './NavigationBar.css';

const NavigationBar = () => (
  <Navbar className="nav-container">
    <Navbar.Header>
      <Navbar.Brand>
        <a href="/"><img src="https://cdn.iconscout.com/public/images/icon/free/png-512/pied-piper-logo-3b6d62b790130fec-512x512.png"/></a>
      </Navbar.Brand>
    </Navbar.Header>
  </Navbar>
)

export default NavigationBar;