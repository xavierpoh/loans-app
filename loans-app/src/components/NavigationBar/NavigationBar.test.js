import React from 'react';
import { shallow } from 'enzyme';
import NavigationBar from './NavigationBar';

describe('NavigationBar', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<NavigationBar />);
    expect(wrapper).toMatchSnapshot();
  })
})