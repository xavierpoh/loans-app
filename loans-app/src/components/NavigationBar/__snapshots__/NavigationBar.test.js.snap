// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBar renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavigationBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "bsStyle": "default",
      "children": <NavbarHeader>
        <NavbarBrand>
                <a
                        href="/"
                >
                        <img
                                src="https://cdn.iconscout.com/public/images/icon/free/png-512/pied-piper-logo-3b6d62b790130fec-512x512.png"
                        />
                </a>
        </NavbarBrand>
</NavbarHeader>,
      "className": "nav-container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <NavbarBrand>
          <a
                    href="/"
          >
                    <img
                              src="https://cdn.iconscout.com/public/images/icon/free/png-512/pied-piper-logo-3b6d62b790130fec-512x512.png"
                    />
          </a>
</NavbarBrand>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <a
            href="/"
>
            <img
                        src="https://cdn.iconscout.com/public/images/icon/free/png-512/pied-piper-logo-3b6d62b790130fec-512x512.png"
            />
</a>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              src="https://cdn.iconscout.com/public/images/icon/free/png-512/pied-piper-logo-3b6d62b790130fec-512x512.png"
/>,
            "href": "/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "src": "https://cdn.iconscout.com/public/images/icon/free/png-512/pied-piper-logo-3b6d62b790130fec-512x512.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "a",
        },
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "bsStyle": "default",
        "children": <NavbarHeader>
          <NavbarBrand>
                    <a
                              href="/"
                    >
                              <img
                                        src="https://cdn.iconscout.com/public/images/icon/free/png-512/pied-piper-logo-3b6d62b790130fec-512x512.png"
                              />
                    </a>
          </NavbarBrand>
</NavbarHeader>,
        "className": "nav-container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <NavbarBrand>
            <a
                        href="/"
            >
                        <img
                                    src="https://cdn.iconscout.com/public/images/icon/free/png-512/pied-piper-logo-3b6d62b790130fec-512x512.png"
                        />
            </a>
</NavbarBrand>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <a
              href="/"
>
              <img
                            src="https://cdn.iconscout.com/public/images/icon/free/png-512/pied-piper-logo-3b6d62b790130fec-512x512.png"
              />
</a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                src="https://cdn.iconscout.com/public/images/icon/free/png-512/pied-piper-logo-3b6d62b790130fec-512x512.png"
/>,
              "href": "/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "src": "https://cdn.iconscout.com/public/images/icon/free/png-512/pied-piper-logo-3b6d62b790130fec-512x512.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "a",
          },
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
